flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/batch/BatchPhysicalExpand.scala

```diff
@@ -53,6 +54,7 @@ class BatchPhysicalExpand(
 
   override def translateToExecNode(): ExecNode[_] = {
     new BatchExecExpand(
+      ShortcutUtils.unwrapConfig(this),
```

Comment:
This will not give you planner config but table config. Maybe we should rename this method. I think we can also retrieve the planner config from a `RelNode`.